internal/app/tfsec/custom/processing.go

```diff
@@ -0,0 +1,75 @@
+package custom
+
+import (
+	"fmt"
+	"github.com/tfsec/tfsec/internal/app/tfsec/parser"
+	"github.com/tfsec/tfsec/internal/app/tfsec/scanner"
+)
+
+func processFoundChecks(checks ChecksFile) {
+	for _, customCheck := range checks.Checks {
+		go func(customCheck Check) {
```

Comment:
I think you were saying earlier about removing this `go`? Doesn't really matter either way though :+1: 
```suggestion
		func(customCheck Check) {
```